client/src/components/right/results/index.js

```diff
@@ -42,13 +41,12 @@ class Results extends React.Component {
         >
           {
             this.props.data.map( (recipe, i) => {
-              console.log(recipe);
               return (
                 <GridTile
                   key={i}
                   title={recipe.fullData.name}
                   subtitle={
-                    <span >Source:
+                    <span>Source:
```

Comment:
@nealfwilliams Lets not have "Source:" in the subtitle. Can we just do this?

`subtitle={
                    <span >
                      <a
                        href={recipe.fullData.source.sourceRecipeUrl}
                        style={{color: 'white', marginLeft: '2px'}}
                        target="_blank"
                      >
                        { recipe.abridgedData.sourceDisplayName }
                      </a>
                    </span>
                  }`